src/group/components/GroupsHomeCard.js

```diff
@@ -98,18 +106,13 @@ const GroupsHomeCard = ({ groups }) => {
       >
         {t('group.home_title')}
       </Typography>
-      <List aria-describedby="groups-list-title">
+      <List
+        aria-describedby="groups-list-title"
+        sx={{ paddingLeft: theme.spacing(2), paddingRight: theme.spacing(2) }}
```

Comment:
Suggestion: You can do this inside the `sx` property and it should be the same
```suggestion
        sx={{ pl: 2, pr: 2 }}
```